MobileFilter/sections/specific_web.txt

```diff
@@ -789,7 +788,7 @@ nordot.app##.main__adWrapper
 nordot.app##.page__ad:not(.page__ad--discoverMore)
 m.youtube.com##.ytp-ad-action-interstitial
 n-tv.de##.topbanner-mobile
-m.arukikata.co.jp,setn.com,hateblo.jp,hatenablog.com,hatenablog.jp,hatenadiary.com,hatenadiary.jp,hatenadiary.org,syosetu.org##.ad
+gameranbu.jp ,seiga.nicovideo.jp,m.arukikata.co.jp,setn.com,hateblo.jp,hatenablog.com,hatenablog.jp,hatenadiary.com,hatenadiary.jp,hatenadiary.org,syosetu.org##.ad
```

Comment:
Unnecessary white space is inserted here.

Reply:
sorry. Fixed it.